packages/browser-sdk/src/feature/features.ts

```diff
@@ -71,9 +72,11 @@ export type FetchedFeature = {
 
 const FEATURES_UPDATED_EVENT = "featuresUpdated";
 
+/**
+ * @internal
+ */
 export type FetchedFeatures = Record<string, FetchedFeature | undefined>;
 
-// todo: on next major, come up with a better name for this type. Maybe `LocalFeature`.
 export type RawFeature = FetchedFeature & {
```

Comment:
I couldn't come up with something better here. I also think we renamed the other type to "FetchedFeature" since this comment was written, so hopefully this is OK now.